res/layout/audio_view.xml

```diff
@@ -96,4 +96,14 @@
 
     </LinearLayout>
 
-</merge>
\ No newline at end of file
+    <LinearLayout
```

Comment:
this is the hack you told  me?

can you please add a `<!-- comment -->` above this tag to explain the rough ongoings? or a link that explains them?

Reply:
added a comment